.circleci/config.yml

```diff
@@ -2,7 +2,42 @@ version: 2.1
 orbs:
   rust: circleci/rust@1.6.0
   win: circleci/windows@2.2.0
+executors:
+  docker-publisher:
+    environment:
+      IMAGE_NAME: ogenev/trin
+    docker:
+      - image: cimg/rust:1.62.0
 jobs:
+  docker-build:
+    executor: docker-publisher
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: docker build -t $IMAGE_NAME:latest .
```

Comment:
`latest` is the tag that we use for our testnet nodes (though it's `ethpm/trin:latest` not `ogenev/trin:latest`. But I do think we'll want to introduce a `stable` or `testnet` tag, since it's not ideal to update every testnet node each time a new commit is pushed to master